en/performance/container-tuning.html

```diff
@@ -111,7 +111,7 @@ <h2 id="jvm-tuning">JVM Tuning</h2>
 <pre>
 &lt;container id="default" version="1.0"&gt;
   &lt;nodes&gt;
-    &lt;jvm gc-options="-Xms4g -Xmx4g -XX:+UseG1GC -XX:MaxTenuringThreshold=15 -XX:NewRatio=1 -XX:+PrintGC" options="-XX:+PrintCommandLineFlags" /&gt;
+    &lt;jvm options="-Xms4g -Xmx4g -XX:+PrintCommandLineFlags" gc-options="-XX:+UseG1GC -XX:MaxTenuringThreshold=15 -XX:NewRatio=1 -XX:+PrintGC" /&gt;
```

Comment:
-XX:+PrintGC is a general option independent of GC algorithm and should be in options rather than gc-options.
I think that most of the arguments pertaining to a certain GC algorithm contains the algorithm in the name.
See fx https://docs.oracle.com/javase/10/gctuning/garbage-first-garbage-collector-tuning.htm#JSGCT-GUID-70E3F150-B68E-4787-BBF1-F91315AC9AB9
Perhaps use some G1 specific options as examples to make a point.

Reply:
Thanks, example updated